siliconcompiler/client.py

```diff
@@ -309,17 +309,25 @@ def fetch_results_request(chip):
             elif resp.status_code == 303:
                 redirect_url = resp.headers['Location']
                 can_redirect = True
-            else:
+            elif resp.status_code == 200:
                 shutil.copyfileobj(resp.raw, zipf)
-                return
+                return 0
```

Comment:
Invert the logic to be True on success and false on Failure

Use True/False instead of 0/1

Reply:
0/1 are more common error code responses. 0 = no error, nonzero = error.

Reply:
True/False